src/bitmessagemain.py

```diff
@@ -42,10 +42,15 @@
     adjustHalfOpenConnectionsLimit, start_proxyconfig)
 from inventory import Inventory
 # Network objects and threads
-from network import (
-    BMConnectionPool, Dandelion, AddrThread, AnnounceThread, BMNetworkThread,
-    InvThread, ReceiveQueueThread, DownloadThread, UploadThread
-)
+from network.addrthread import AddrThread
+from network.connectionpool import BMConnectionPool
+from network.dandelion import Dandelion
+from network.announcethread import AnnounceThread
+from network.networkthread import BMNetworkThread
+from network.invthread import InvThread
+from network.receivequeuethread import ReceiveQueueThread
+from network.downloadthread import DownloadThread
+from network.uploadthread import UploadThread
```

Comment:
You are binding the code to pathmagic harder instead of preparing its removal ):

`network` is a package, so you can write `from .network import ..` ant it will work both in python2.7 and python3 in installed and portable mode.

Reply:
Thanks it wasn't very clear to me what the best way to fix imports for python 3 would be. I'll give .network a try.